tools/build-docs

```diff
@@ -1,3 +1,18 @@
-#!/usr/bin/env bash
-set -eu
-make -C "$(dirname "$0")"/../docs clean html
+#!/usr/bin/env python3
+
+# check for the venv
+from lib import sanity_check
+
+sanity_check.check_venv(__file__)
+
+import os
+import subprocess
+
+
+def main() -> None:
+    path = os.path.join(os.path.dirname(__file__), "../docs")
```

Comment:
Can you use the same path access conventions that our other tools do, e.g. `ZULIP_PATH`?  It's nice to have standard conventions for this sort of thing.

Reply:
Sure